service/tests/preludeTiny.repl

```diff
@@ -1,5 +1,7 @@
 //import CanCan = "ai7t5-aibaq-aaaaa-aaaaa-c";
 import CanCan = "rrkah-fqaaa-aaaaa-aaaaq-cai";
+// assume the same identity as a `dfx call` command
+identity default "~/.config/dfx/identity/default/identity.pem";
```

Comment:
analogus change to https://github.com/dfinity/cancan/pull/21/commits/aa15e4ba844037e66bdabf8fee6438e0654b82c1#r636396175